spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/MetricsProperties.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for namespace 'spring.cloud.dataflow.metrics'.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+@ConfigurationProperties(prefix = MetricsProperties.PREFIX)
+public class MetricsProperties {
+
+	public static final String PREFIX = "spring.cloud.dataflow.metrics";
+
+	private Collector collector = new Collector();
+
+	public Collector getCollector() {
+		return collector;
+	}
+
+	public void setCollector(Collector collector) {
+		this.collector = collector;
+	}
+
+	public static class Collector {
+		private String url;
```

Comment:
Can this be more specific type to denote the URL (just to have some URI scheme validation etc.,)?